1-D_Array/Python/Merge_Two_Sorted_Arrays.py

```diff
@@ -0,0 +1,37 @@
+#Python program to merge two sorted array a[] and b[] and storing it in c[]
```

Comment:
No import statements required in any file?

Reply:
Nope